ci-operator/step-registry/redhat-chaos/container-scenarios/etcd-hangup/redhat-chaos-container-scenarios-etcd-hangup-commands.sh

```diff
@@ -17,6 +17,7 @@ echo "kubeconfig loc $$KUBECONFIG"
 
 export KRKN_KUBE_CONFIG=$KUBECONFIG
 export NAMESPACE=$TARGET_NAMESPACE
+export ENABLE_ALERTS=False
```

Comment:
@chaitanyaenr I turned off alerting for this type of execution because there is a high chance that the leader will change and the alert was always firing. Should we consider adding a second list of alerts to cover the other alerts but take out the leader change for this scenario. Could also be a future work item

Reply:
See most recent run here passed: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/39752/rehearse-39752-periodic-ci-redhat-chaos-krkn-hub-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa/1663669406187655168

Reply:
Think we should consolidate on our alerts - it might be better to only have metrics that describe the overall health and performance of the components/cluster to avoid things we are hitting here with leader elections. For instance, we can check on etcd performance and if leader exists vs leader changes to avoid wrong alarms like you mentioned @paigerube14.